src/main/resources/edu/cornell/kfs/sys/cu-spring-sys.xml

```diff
@@ -203,6 +203,7 @@
 
     <bean id="autoDisapproveService" class="edu.cornell.kfs.sys.batch.service.impl.CuAutoDisapproveDocumentsServiceImpl" parent="autoDisapproveService-parentBean">
         <property name="routeHeaderService" ref="enDocumentRouteHeaderService" />
+        <property name="personService" ref="personService" />
```

Comment:
I believe the parent bean is already injecting this service, so it shouldn't be necessary to inject it again here.